### PR TITLE
ping graph fix & control changes

### PR DESCRIPTION
**Pull request by maxomatic458**
fixed the ping bar-chart (latest values now on the right)
exiting playit with ``ctrl-c`` clears the screen
added keys 1 - 4 to navigate the Ui tabs

(first pull request i hope i did it right)

https://user-images.githubusercontent.com/104733404/186985271-ff52ad7f-efe0-4bdc-ab73-8c762bab242f.mp4

### CODE REVIEW

File: packages/agent_cli/src/graphics.rs [hunk 1]
```diff
@@ -125,6 +125,7 @@ impl GraphicInterface {
             match event {
                 Some(Event::Key(key)) => {
                     if key.code == KeyCode::Char('c') && key.modifiers.contains(KeyModifiers::CONTROL) {
+                        print!("\x1B[2J\x1B[1;1H");
```
Comment (loriopatrick) [inline packages/agent_cli/src/graphics.rs:128]: Is this cross platform?

Comment (maxomatic458) [reply packages/agent_cli/src/graphics.rs:128]: worked in linux subsystem and as .exe (but i guess people will just hit the x on windows)

Comment (loriopatrick) [reply packages/agent_cli/src/graphics.rs:128]: cool, let's move this to the GraphicInterface deconstructor and let's add a comment about what it does. If you got the snippet from somewhere else let's add the link to that source in the comment too.

https://github.com/playit-cloud/playit-agent/blob/7030675c86498a505fda6a9bdbc96eccc6790cac/packages/agent_cli/src/graphics.rs#L87

Comment (maxomatic458) [reply packages/agent_cli/src/graphics.rs:128]: ok,
sorry if im a bit nooby but i can just edit the files inside this pull request then?

Comment (loriopatrick) [reply packages/agent_cli/src/graphics.rs:128]: Yes, you should just be able to update your branch and push a new commit.

Comment (maxomatic458) [reply packages/agent_cli/src/graphics.rs]: just moved it,
``print!("\x1B[2J\x1B[1;1H")`` does not "clear" the terminal, it scrolls down and resets the cursor pos, so it doesn't delete any previous commands.
``print!("{esc}c", esc = 27 as char);`` (also cross platform) would clear the terminal fully, but i don't think there really is the need for it.

**Pull request closed without merging**